workshops/workshops.js

```diff
@@ -0,0 +1,49 @@
+import { checkAuth, deleteParticipant, getWorkshops, logout } from '../fetch-utils.js';
+
+checkAuth();
+
+const logoutButton = document.getElementById('logout');
+
+logoutButton.addEventListener('click', () => {
+    logout();
+});
+
+async function renderWorkshops() {
+    const container = document.getElementById('workshop-container');
+    container.textContent = '';
+    const workshops = await getWorkshops();
+    
+    for (const workshop of workshops) {
+        container.append(renderWorkshop(workshop));
+    }
+}
+
+
+function renderWorkshop(object) {
```

Comment:
Similar deal here with the passed in argument as my comment in your fetch-utils. What specifically is the object? Is passing in an object necessary for this render function? When you call renderWorkshop on line 48 you aren't passing in any arguments. Just be sure to be consistent.